KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment2/StockServiceFactory.java

```diff
@@ -0,0 +1,18 @@
+
+package edu.KaylaKornelis.advancedjava.Assignment2;
+
+/**
+ *
+ * @author Kornelis
+ */
+public class StockServiceFactory {
+    
+    public StockService getStockService(StockService stockServiceType){
```

Comment:
should be:
public static StockService getStockService() 
No args. 

I know you will find examples of factory patterns that let the caller specify the type of thing they want, but that's a broken factory pattern. 

You want the actual implementation of stock service to be hidden from code using it.

Reply:
I removed the argument

Reply:
@smarks Should I just have the getStockService method return a new instance of BasicStockService since it's the only option at the moment? And remove the if conditional?